src/tests/vld_main/vld_main.cpp

```diff
@@ -0,0 +1,48 @@
+// vld_main.cpp : Defines the entry point for the console application.
+//
+
+#include "stdafx.h"
+#include <windows.h>
+#include <malloc.h>
+
+#define VLD_FORCE_ENABLE
+#include <vld.h>
+
+
+class MemoryLeak {
+public:
+    MemoryLeak(size_t n) { l = malloc(n); }
+    ~MemoryLeak() { free(l); }
+private:
+    void* l;
+};
+
+static void* s_m = malloc(10);
+static char* s_n = new char[20];
+
+static MemoryLeak* pml = new MemoryLeak(70); // leaks a new pointer and malloc(70)
+static MemoryLeak ml{ 80 }; // *should* be freed and not report as a memory leak
+
+void* g_m = malloc(30);
+char* g_n = new char[40];
+
+
+int _tmain(int argc, _TCHAR* argv[])
+{
+    void* m = malloc(50);
+    char* n = new char[60];
+
+    return VLDGetLeaksCount() == 8 ? 0 : 1;
```

Comment:
VS2015 ? Debug / Release ?
AppVeyor reports 8 leaks in the Release test for both Win32 and x64.
Can you post the full report so we can check if they are crt leaks